terrapkgs.nix

```diff
@@ -0,0 +1,5 @@
+(import (builtins.fetchTarball {
```

Comment:
Please just take this entire file and embed it in lib.nix instead of making it seperate.